src/pprint.rs

```diff
@@ -108,7 +108,11 @@ fn pprint_inner<'a, Tgt, Aux>(
             }
             PrintMode::Compact => {
                 let joined = imp.aux().extra_column_values().join(", ");
-                r = row![format!("{indent_str}/* {joined} */\n{main_str}\n")];
+                if !joined.is_empty() {
+                    r = row![format!("{indent_str}/* {joined} */\n{main_str}\n")];
+                } else {
+                    r = row![main_str];
+                }
```

Comment:
This is for omitting unnecessary `/* */` when there are no extra columns.

Reply:
As above, if you break this into another PR, happy to merge into main immediately.